Netable/Netable/DelayedOperations.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  DelayedOperations.swift
+//  Netable
+//
+//  Created by Nigel Brooke on 2020-06-30.
+//  Copyright © 2020 Steamclock Software. All rights reserved.
+//
+
+import Foundation
+
+// Utility class for scheduling future operations that can easily be identified and cancelled later
+// Used to hold in flight requests that are waiting for a retry.
+internal class DelayedOperations {
+    private var operations: [(timer: Timer, id: String)] = []
```

Comment:
Is there any difference here between using a tuple or struct here?
I'm mostly just curious, I don't think it matters either way since this is an internal class.

Reply:
Less typing, since it's only used in one place, but not much difference other than that.